src/components/Feed/ProposalCard.js

```diff
@@ -31,14 +32,17 @@ function Card({
 }) {
   const theme = useTheme()
   const history = useHistory()
+  const {
+    connectedDAO: { address: daoAddress },
+  } = useDAO()
 
   const { below } = useViewport()
 
   const handleSelectProposal = useCallback(() => {
     const entityPath =
       proposal.type === ProposalTypes.Decision ? 'vote' : 'proposal'
-    history.push(`/${entityPath}/${proposal.number}`)
-  }, [history, proposal.number, proposal.type])
+    history.replace(`/${daoAddress}/${entityPath}/${proposal.number}`)
```

Comment:
Same comment here. 

What's more i would like to consider not having the `{daoId}/home` route and just leave it as `/{daoId}`.
This way we would add the proposal path to the current path `history.push(${history.location.pathname}/${entityPath}/${proposal.number})`

Reply:
yeah make sense maybe was for that that i changed for replace